packages/host/tests/acceptance/catalog-app-test.gts

```diff
@@ -216,96 +218,66 @@ module('Acceptance | catalog app tests', function (hooks) {
     assert.strictEqual(message.content.body, expectedMessage);
   }
 
-  async function verifyInstanceExists(
-    targetRealm: string,
-    targetDirPrefix: string,
-    dirName: string,
-    assert: Assert,
-  ) {
-    await visitOperatorMode({
-      submode: 'code',
-      fileView: 'browser',
-      codePath: `${targetRealm}index`,
-    });
-    await waitForCodeEditor();
-
-    await waitFor(`[data-test-directory^="${targetDirPrefix}"]`);
-    const element = document.querySelector(
-      `[data-test-directory^="${targetDirPrefix}"]`,
-    );
-    const fullPath = element?.getAttribute('data-test-directory');
-    await click(`[data-test-directory="${fullPath}"]`);
-
-    assert.dom(`[data-test-directory="${fullPath}"] .icon`).hasClass('open');
-
-    const instancePath = `${fullPath}${dirName}/`;
-    await waitFor(`[data-test-directory="${instancePath}"]`);
-    await click(`[data-test-directory="${instancePath}"]`);
-
-    assert
-      .dom(`[data-test-directory="${instancePath}"] .icon`)
-      .hasClass('open');
+  async function verifySubmode(assert: Assert, submode: Submode) {
+    assert.dom(`[data-test-submode-switcher=${submode}]`);
```

Comment:
what specifically are we asserting here? that it exists? if so you wrote the assertion incorrectly